web/src/main/java/org/springframework/security/web/FilterChainProxy.java

```diff
@@ -147,6 +154,12 @@ public class FilterChainProxy extends GenericFilterBean {
 
 	private static final String FILTER_APPLIED = FilterChainProxy.class.getName().concat(".APPLIED");
 
+	static final String REQUEST_OBSERVATION_NAME = "http.requests";
```

Comment:
If this is for the whole request, Spring MVC already covers this, it is called `http.server.requests`.

Reply:
For simplicity, I ultimately decided to leave the wrapping observation out. We can always add it later if needed.